libs/template/config.go

```diff
@@ -143,6 +152,44 @@ func (c *config) skipPrompt(p jsonschema.Property, r *renderer) (bool, error) {
 	return true, nil
 }
 
+func (c *config) promptOnce(property *jsonschema.Schema, name, defaultVal, description string) error {
+	var userInput string
+	var err error
```

Comment:
Why is it defined at top level?

Reply:
Needed because we use `userInput, err = ...` and `c.values[name], err = ...`. `:=` is not an option because the first option exists beforehand in both cases. Moved the declaration closer to where it's used to make this clearer.